demo/site/src/pages/[[...path]].tsx

```diff
@@ -1,20 +1,20 @@
 import { defaultLanguage, domain } from "@src/config";
-import { GQLPage } from "@src/graphql.generated";
+import { GQLPageTreeNodeScopeInput } from "@src/graphql.generated";
 import NotFound404 from "@src/pages/404";
 import PageTypePage, { loader as pageTypePageLoader } from "@src/pageTypes/Page";
 import createGraphQLClient from "@src/util/createGraphQLClient";
-import { gql } from "graphql-request";
+import { gql, GraphQLClient } from "graphql-request";
 import { GetStaticPaths, GetStaticProps, InferGetStaticPropsType } from "next";
 import { ParsedUrlQuery } from "querystring";
 import * as React from "react";
 
 import { GQLPagesQuery, GQLPagesQueryVariables, GQLPageTypeQuery, GQLPageTypeQueryVariables } from "./[[...path]].generated";
 import { PreviewData } from "./api/site-preview";
 
-type PageProps = GQLPage & {
+type PageProps = {
     documentType: string;
     id: string;
-};
+} & Record<string, unknown>;
```

Comment:
GQLPage is wrong as it might be any other page type. `Record<string, unknown>` allows the object spread in [line 76](https://github.com/vivid-planet/comet/pull/1815/files#diff-f7b50ba4b75963b57ce596aa9140dcba17782d38fd041736f1990350b6ad4d4eR76)